sktime/transformations/compose.py

```diff
@@ -171,7 +171,7 @@ def __init__(self, steps):
         self._anytagis_then_set("scitype:instancewise", False, True, ests)
         self._anytagis_then_set("fit_is_empty", False, True, ests)
         self._anytagis_then_set("transform-returns-same-time-index", False, True, ests)
-        self._anytagis_then_set("skip-inverse-transform", True, False, ests)
+        self._anytagis_then_set("skip-inverse-transform", False, True, ests)
```

Comment:
for reviewers: old = "skip inverse iff at least one component skips the inverse" -> new: "skip inverse iff all components skip the inverse"